am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -165,7 +169,13 @@ public void revokeResourceAccess(@NotNull @Valid ExplicitAccessMetadata accessMe
         + "and permissions {{result.access.permissions}}")
     public FilteredResourceEnvelope filterResource(@NotBlank String userId,
                                                    @NotEmpty Set<@NotBlank String> userRoles,
-                                                   @NotNull @Valid Resource resource) {
+                                                   @NotNull @Valid Resource resource,
+                                                   @NotEmpty @Valid Map<JsonPointer, SecurityClassification>
+                                                       attributeSecurityClassification) {
+
+        if (attributeSecurityClassification.get(JsonPointer.valueOf("")) == null) {
```

Comment:
May be you can use Lombok.checkNotNull method rather than using hard coded null. Or even Guava Preconditions

Reply:
done